src/components/pacbio/PacbioLibraryCreate.vue

```diff
@@ -17,7 +17,7 @@
       :static="isStatic"
       scrollable
     >
-      <b-form id="libraryCreateModal">
+      <b-form id="libraryCreateModal" @submit="createLibrary">
```

Comment:
Move the submit action from the button onto the form in order to allow the native validation to trigger.